justfile

```diff
@@ -80,7 +80,6 @@ _test-node-rollup command="":
     pnpm run --filter rollup-tests test{{ command }}
 
 # Fix formatting issues both for Rust, Node.js and all files in the repository
-
 fmt: fmt-rust fmt-repo
 
 fmt-rust:
```

Comment:
You should remove `cargo shear --fix` under `fmt-rust`, and recover it in `fix-rust`